sensum-pwa/src/ui/elements/Navbar.res

```diff
@@ -0,0 +1,19 @@
+@react.component
+let make = (~leftComponent: option<React.element>=?) => {
+  switch leftComponent {
+  | None => 
+    <div
+      className="fixed w-full bg-purple-900 border-purple-600 border-2">
+      <h1 className="pl-2 text-4xl text-purple-50">
+        {"sensum"->React.string}
+      </h1>
+    </div>
+  | Some(component) =>
+    <div className="fixed w-full bg-purple-900 border-purple-600 border-2 flex flex-row-reverse">
+      <h1 className="fixed top-0 left-2 right-0 text-4xl text-purple-50">
+        {"sensum"->React.string}
+      </h1>
+      {component}
+    </div>
```

Comment:
I think this is little tangled, why no go with a "classic _flexbox_" solution with `flex `and `justify-between`? I also added previous suggestion fix.

```suggestion
    <header className="fixed w-full bg-purple-900 border-purple-600 border-b-2 flex justify-between">
      <h1 className="text-4xl text-purple-50">
        {"sensum"->React.string}
      </h1>
      {component}
    </div>
```

Reply:
totally agree! it was just because I'm rusty with css and flexbox.